doc/source/user_guide/enhancingperf.rst

```diff
@@ -812,10 +812,11 @@ you have an expression--for example
 
 .. ipython:: python
 
-   df = pd.DataFrame({'strings': np.repeat(list('cba'), 3),
-                      'nums': np.repeat(range(3), 3)})
+   df = pd.DataFrame(
+       {"strings": np.repeat(list("cba"), 3), "nums": np.repeat(range(3), 3)}
+   )
    df
-   df.query('strings == "a" and nums == 1')
+   df.query("strings == "a" and nums == 1")
```

Comment:
We need to use single quotes inside the double quotes

Reply:
doc ci seems to solve， but "pandas-dev.pandas (Linux py38_np_dev)" failed, Is this normal?

Reply:
That's unrelated, it's a flakey build